style.css

```diff
@@ -103,3 +104,85 @@ p {
     color:#ffffff;
     text-align-last: center;
 }
+/* touch menu*/
+._touch .bottom{
+    position: absolute;
+    bottom: 10px;
+    left: 120px;
+    display: block;
+    width: 0;
+    height: 0;
+    margin: 0 0 0 5px;
+    transition: transform 0.3s ease 0s;
+    border-left: 50px solid transparent;
+    border-right: 50px solid transparent;
+    border-top: 100px solid #fff;
+    z-index: 1;
+    opacity: 20%;
```

Comment:
There's a lot of copy pasted css code:

![image](https://user-images.githubusercontent.com/6686679/215521463-8a3b755b-0df5-4aec-9d80-364e42902c00.png)

You can create a new class, which will have this styles, and use it for each arrow.
This way you don't need copy-paste:
```
.mobile-btn {
  position: absolute;
  bottom: 10px;
  display: block
  width: 0;
  height: 0;
  transition: transform 0.3s ease 0s;
  z-index: 1;
  opacity: 20%;
}

.left {
    left: 10px;
    border-bottom: 50px solid transparent;
    border-right: 100px solid #fff;
    border-top: 50px solid transparent;
}
```

And in html use **both** classes, to apply **both** styles to the *same* element:
```
<div class="mobile-btn left">Left button</div>
```

This `div` will have styles from `.mobile-btn` and from `.left`.

You can use `.mobile-btn` for all buttons:
```
<div class="mobile-btn left">Left button</div>
<div class="mobile-btn right">Right button</div>
<div class="mobile-btn forward">Forward button</div>
<div class="mobile-btn back">Back button</div>
```

So `.mobile-btn` class will have *shared* styles (styles which are the same for all the buttons).
Classes `.left`, `.right`, `.forward`, `.back` will have *styles, which are different for each button*.